wdio.conf.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const localIP = require('ip');
+const path = require('path');
+
+const staticServerPort = 8080;
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  specs: [
+    path.join(__dirname, 'packages', '**', 'tests', 'specs', '**'),
+  ],
+  staticServerPort,
+  staticServerLog: false,
+  staticServerFolders: [
+    { mount: '/', path: path.join(__dirname, 'packages', 'terra-site', 'dist') },
+  ],
+
+  suites: {
+    badge: [
+      path.join(__dirname, 'packages', 'terra-badge', 'tests', 'specs', '**'),
+    ],
+  },
+
+  seleniumDocker: {
+    enabled: !process.env.TRAVIS,
+    cleanup: false,
+  },
+  // eslint-disable-next-line global-require
+  webpackConfig: require('./packages/terra-site/webpack.config.js'),
+};
+
+
+config.services = wdioConf.config.services.concat(['webpack', 'static-server']);
```

Comment:
I know @emilyrohrbough was looking into getting webpack-dev-server working rather than using webpack to generate the site.  I think I'd ideally like to use webpack-dev-server if possible.  Should we wait for that?

Reply:
👍 I'm awaiting her pull request!!!

Reply:
What's the status on this?